pkg/virtctl/vmexport/vmexport.go

```diff
@@ -144,6 +144,8 @@ var (
 	manifestOutputFormat string
 )
 
+const downloadRetries = 2
```

Comment:
Okay, would we want to retry every time the download fails or just when the server returns a bad status (as originally intended)? I prefer the second but since we'll include a flag to explicitly request retries maybe it makes sense to retry even when other kind of error occurs.

Reply:
Yeah I think it only makes sense to retry when the server returns bad status.

Reply:
I think default retries should be 0 what do you think @alicefr?

Reply:
I can use 0 in vmexport if you prefer, but for the memory dump integration, I think we should at least use 1 to mitigate the bug.

Reply:
Makes sense, left the higher default retries in the memory dump integration and justified it with a comment.